syntax = "proto3";

service TranscriptionService {
  rpc TranscribeFile (TranscriptionRequest) returns (TranscriptionResponse) {}
}

message TranscriptionRequest {
  int32 id = 1;
  bytes audio = 10;
}

message TranscriptionResponse {
  int32 id = 1;
  string text = 10;
}